tests/unit/services.py

```diff
@@ -21,37 +22,11 @@
 from lms.services.oauth1 import OAuth1Service
 from lms.services.oauth2_token import OAuth2TokenService
 from lms.services.oauth_http import OAuthHTTPService
+from lms.services.async_oauth_http import AsyncOAuthHTTPService
 from lms.services.user import UserService
 from lms.services.vitalsource import VitalSourceService
 from tests import factories
 
-__all__ = (
-    # Meta fixture for creating service fixtures
-    "mock_service",
-    # Individual services
-    "application_instance_service",
-    "assignment_service",
-    "blackboard_api_client",
-    "canvas_api_client",
-    "canvas_service",
-    "course_service",
-    "file_service",
-    "grading_info_service",
-    "grant_token_service",
-    "group_info_service",
-    "grouping_service",
-    "h_api",
-    "http_service",
-    "launch_verifier",
-    "lti_h_service",
-    "lti_outcomes_client",
-    "oauth1_service",
-    "oauth2_token_service",
-    "oauth_http_service",
-    "user_service",
-    "vitalsource_service",
```

Comment:
I needed to add `async_oauth_http_service` here but I don't see why we should keep this list, happy to add it back if this would break something.

Reply:
This is actually need because we do `from tests.unit.services import *`:

https://github.com/hypothesis/lms/blob/f677d836efb068f7365a961fbadc0836c0c10bac/tests/unit/conftest.py#L13

It's necessary to import all the fixtures from `services.py` into `conftest.py` otherwise pytest won't discover those fixtures.

Without the `__all__` `conftest.py` is now importing a bunch more stuff (e.g. everything that `services.py` imports) which is wrong even if it doesn't actually cause any tests to fail.

I think this is over-organization and we should just inline the contents of `tests/unit/services.py` into `tests/unit/conftest.py`. I think it's fine to just keep all pytest fixtures in the standard location in the `conftest.py` files where people (and pytest_ expect to find them. Some of the `conftest.py` files might get quite big but that's probably fine.

Reply:
Oh I forgot about fixture discovery, fair enough, added it back :+1: